<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>750</width>
     <height>580</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>750</width>
     <height>580</height>
    </size>
   </property>
   <widget class="QPlainTextEdit" name="messageBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>730</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>730</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>730</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="chatMessages">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>570</width>
      <height>460</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>570</width>
      <height>460</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>570</width>
      <height>460</height>
     </size>
    </property>
   </widget>
   <widget class="QListWidget" name="usersList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>150</width>
      <height>460</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>460</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>460</height>
     </size>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="serverLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>60</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
